backend/lib/dbHelpers.js

```diff
@@ -1,7 +1,11 @@
+// node modules
 const Guid = require('guid')
+const Bluebird = require('bluebird')
+// local modules
+const utils = require('./utils.js')
 
 module.exports = (client) => {
```

Comment:
Were you going for a pattern like the one we used in the [tasminions blog](https://github.com/tasminions/blog/blob/master/lib/handlers.js)? What is the thinking behind writing this as a constructor function instead of a function that returns an object with methods? Was it because some of the methods reference each other? You can fix this by using ES5 functions (their contexts are automatically bound to the object they are attached to). Like this:
```javascript
const foo = (_) => {
  return {
    add: function (a, b) { return a + b },
    subtract: function (a, b) { return this.add(a, -b) }
  }
}
```